packages/core/src/languages/solidity/types.ts

```diff
@@ -227,6 +227,7 @@ export type SphinxTransactionReceipt = {
   from: string
   gasPrice: string
   gasUsed: string
+  blobGasUsed: string | null
```

Comment:
The `SphinxTransactionReceipt` is a user-facing type because we include it in our deployment artifacts. Could you update `deployment-artifacts.md`?

Reply:
Also, can you update the JSDoc of this type? Specifically, what is the `blobGasUsed`, and when is it included?